### DISCUSSION TITLE
Allow for setting aliases for scripts in the command palette

### DISCUSSION THREAD

**Discussion by YousufSSyed** [johnlindquist/kit#1409] [Ideas]
Instead of writing the full command or script name out, I'd like to set aliases for them, as short as 1-2 characters; and multiple aliases for each script.

**Comment by johnlindquist** [comment]
You can use:

```js
// Trigger: aa
// Alias: bb
```

Trigger will run after you type: "aa"
Alias will run after you type "bb", then spacebar

We don't support multiple aliases
